server/etcdserver/server.go

```diff
@@ -107,6 +107,8 @@ const (
 	readyPercent = 0.9
 
 	DowngradeEnabledPath = "/downgrade/enabled"
+
+	maxRaftAdvanceDelay = 500 * time.Millisecond
```

Comment:
Where this variable comes from?

Reply:
Will remove this hardcoded timeout value